### DISCUSSION TITLE
React Native support

### DISCUSSION THREAD

**Discussion by Matthew1172** [opensheetmusicdisplay/opensheetmusicdisplay#1174] [Q&A] [answered]
Hello,

Is it possible to use OSMD in React Native? Can anyone share some code/resources/articles on how to achieve this?

Thank you,
Matthew Pecko

**Comment by sschmidTU** [comment, marked as answer]
I'm not an expert on React Native, so I probably can't help much,
but you probably need to use a webview, or maybe something like this:
https://www.npmjs.com/package/react-native-canvas

We actually have a React component (not React Native and outdated, but it might help):
https://github.com/opensheetmusicdisplay/react-opensheetmusicdisplay

there's some more discussion in #491, #432 and #267.

Also, on the Vexflow side of things, see this:
https://github.com/0xfe/vexflow/issues/338#issuecomment-277338739
https://github.com/panarch/standalone-vexflow-context
